src/main/java/org/apache/ibatis/executor/resultset/DefaultResultSetHandler.java

```diff
@@ -643,7 +643,7 @@ private Object createResultObject(ResultSetWrapper rsw, ResultMap resultMap, Res
     if (resultObject != null && !hasTypeHandlerForResultObject(rsw, resultMap.getType())) {
       final List<ResultMapping> propertyMappings = resultMap.getPropertyResultMappings();
       for (ResultMapping propertyMapping : propertyMappings) {
-        // issue gcode #109 && issue #149
```

Comment:
This is a reference to an old Google Code issue - not a typo

Reply:
@Oliverwqcwrw Thank you!  I did run a squash on this one due to number of commits and overall change being simple cleanup.  That may impact you a bit when you rebase back up.